frontend/js/src/common/listens/MBIDMappingModal.tsx

```diff
@@ -163,6 +163,7 @@ export default NiceModal.create(({ listenToMap }: MBIDMappingModalProps) => {
         style={visible ? { display: "block" } : {}}
```

Comment:
Both the classname and style here don't follow the same convention as the other modals, if by any chance it is leftover from trying to get the modal to show...

Reply:
Cleaned up